### ISSUE TITLE
Cloudflare Workers - toBuffer with jpeg fails due to reliance on jpeg-js

### ISSUE THREAD

**Issue by matthewhardern** [image-js/image-js#619]
toBuffer as above fails due to jpeg-js reliance on Buffer.from(), although cloudflare workers can use Buffer it needs to be added with 'node:buffer' package. The png works perfectly, would be nice to work on updating the fast-jpeg and using that to remove this reliance? Awesome library BTW, its been the easiest one to setup on cloudflare workers. Happy to create a PR for docs on usage if anyone would like.

**Comment by stropitek** [comment]
Have you tried opening an issue in the jpeg-js repo? fast-jpeg doesn't read / write jpeg, only metadata so we can't rely on that.

**Comment by targos** [comment]
Yeah, I'm not sure there's anything we can do if the problem is in `jpeg-js`
